code/unclassified/src/average/Average.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
+// Part of Cosmos by OpenGenus Foundation
+class Average
+{
+    /**
+    * @param arr array of integers to get the average of.
+    * @param n number of elements.
+    * @return the average calculated as sum(arr)/n
+    */
+    static double getAverage(ArrayList<Integer> arr)
+    {
+        if (arr.isEmpty()) {
+            return 0.0;
+        } 
+        double sum = 0;
+        for(int value : arr)
+        {
+            sum += value;
+        }
+        
+        return sum / arr.size();
+    }
+    public static void main(String args[])
+    {
+        int n;
+        // int[] arr = new int[1000];
+        ArrayList<Integer> arr = new ArrayList<Integer>();
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Enter elements in array (-1)exit ");
+        
+        while ((n = scan.nextInt()) != -1)
```

Comment:
I meant to move `n` right above this line so it's close to where it's used.

Reply:
Done.